### PR TITLE
Npm updates

### PR DESCRIPTION
**Pull request by harryadel**
Update the npm packages as npm scripts were failing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,27 +33,29 @@
...
     "updater-license": "^1.0.0",
-    "webpack": "^2.7.0",
-    "webpack-dev-server": "^2.4.5"
+    "webpack": "^4.44.1",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-cli": "^3.3.12"
   },
   "peerDependencies": {
-    "cytoscape": "^3.2.0"
+    "cytoscape": "^3.15.2"
```
Comment (maxkfranz) [inline package.json:56]: Is it necessary to update the peer dependency, or would the extension still be compatible with 3.2+?

File: package.json [hunk 2]
```diff
@@ -33,27 +33,29 @@
...
   },
   "dependencies": {
-    "dagre": "^0.8.2"
+    "dagre": "^0.8.5"
```
Comment (maxkfranz) [inline package.json:59]: Updating the version of Dagre is great.  It will require a feature version update to 2.3.0 of this extension.

**Comment by harryadel** [comment]
@maxkfranz Please recheck Max, ty.

**Comment by maxkfranz** [comment]
Looks good.  I'll set aside some time tomorrow to merge and release this.

**Pull request merged**